android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -59,7 +59,7 @@
 	"borderColor", "borderRadius", "borderWidth",
 
 	// layout / dimension (size/width/height have custom accessors)
-	"left", "top", "right", "bottom", "layout", "zIndex", TiC.PROPERTY_CENTER,
+	"left", "top", "right", "bottom", "layout", "zIndex",
```

Comment:
It should probably be pulled out to it's own PR. The newer V8 versions are much pickier about double property definitions so they'll basically abort on Ti.UI.View (and all subclasses) with this, so I needed the fix to get a working version.